handler/handler.go

```diff
@@ -0,0 +1,79 @@
+package handler
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/ONSdigital/dp-search-data-importer/event"
+	"github.com/ONSdigital/dp-search-data-importer/models"
+	"github.com/ONSdigital/log.go/log"
+)
+
+//go:generate moq -out eventtest/result_writer.go -pkg eventtest . ResultWriter
+
+var _ event.Handler = (*BatchHandler)(nil)
+
+// BatchHandler handles batches of publishedContentModel events that contain CSV row data.
+type BatchHandler struct {
+	resultWriter ResultWriter
+}
+
+// TODO : ResultWriter dependency that outputs results
+type ResultWriter interface{}
+
+// NewBatchHandler returns a resultWriter TBD.
```

Comment:
Why TBD?

Reply:
2nd part of this task for integrating with Elastic search - https://trello.com/c/jhN0Yhlc/850-extend-search-data-importer-to-upsertupdate-add-docs-from-elasticsearch-ons-index-part-2

Reply:
I guess it would make sense this was a TODO still ...